sentry-android-gradle-plugin/src/main/groovy/io/sentry/android/gradle/SentryPlugin.groovy

```diff
@@ -101,12 +101,21 @@ class SentryPlugin implements Plugin<Project> {
      * @return
      */
     static Task getProguardTask(Project project, ApplicationVariant variant) {
```

Comment:
I think we should rename this method to `getMinificationTask` as it can return R8-task, not proguard

Reply:
Below there's still stuff like `autoProguardConfig` which is part of the extension type. That's because R8 is just pretending to be proguard so I'd just leave it for the time being.